src/cosmicqc/analyze.py

```diff
@@ -16,12 +16,12 @@
 )
 
 
-def find_outliers(
+def identify_outliers(
     df: pd.DataFrame,
-    metadata_columns: List[str],
     feature_thresholds: Union[Dict[str, float], str],
     feature_thresholds_file: Optional[str] = DEFAULT_QC_THRESHOLD_FILE,
-) -> pd.DataFrame:
+    include_threshold_scores: bool = False,
+) -> Union[pd.Series, pd.DataFrame]:
```

Comment:
What exactly does this function return? I am 100% sure what the Union of a pd.Series and pd.DataFrame means. 

I am mainly asking because it isn't totally clear to me what the difference is between the identify_outliers and find_outliers is.

Reply:
So identify outliers doesn't couple the metadata with the outlier label, it can output just general info like how many outliers identified and such? If so, that makes total sense!

Reply:
Yes, it could be used to count how many outliers were identified using the output and is decoupled from the metadata.